src/web/client/utilities/commonUtil.ts

```diff
@@ -16,16 +16,23 @@ import { schemaEntityName } from "../schema/constants";
 import { telemetryEventNames } from "../telemetry/constants";
 import WebExtensionContext from "../WebExtensionContext";
 import { SETTINGS_EXPERIMENTAL_STORE_NAME } from "../../../client/constants";
-import { decode, encode } from 'js-base64';
 
-// decodes base64 to text
-export function convertfromBase64ToString(data: string) {
-  return decode(data);
+// decodes file content to UTF-8
+export function convertContentToUint8Array(content: string, isBase64Encoded: boolean): Uint8Array {
+    if(isBase64Encoded){
+        return new Uint8Array(Buffer.from(content, BASE_64));
+    }
+
+    return new TextEncoder().encode(content as string);
```

Comment:
It can be single line
return isBase64Encoded? new Uint8Array(Buffer.from(content, BASE_64)): new TextEncoder().encode(content as string);